packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -626,6 +630,9 @@ export default defineComponent({
       }
 
       const errorObject = formatErrorMessageToObject(error.message)
+      if (this.errors[item.meta.uploadId]?.statusCode === 423) {
+        return this.$gettext('Upload folder is locked')
```

Comment:
@kulmann @tbsbdr Suggestions for text? How about `The folder you're uploading to is locked`?

Reply:
yep, works for me 👍